HOWTOBUILD.md

```diff
@@ -0,0 +1,35 @@
+## How to Build
+### Clone this repository
+```
+$ git clone https://github.com/euphony-io/euphony.git
+$ cd euphony/
+```
+### Build from the command line
+```
+$ ./gradlew build
+```
```

Comment:
As you know, Windows's cmd couldn't understand `./` as the location.
(Windows Powershell, and linux bash, Mac OS are working well)

how about simple description for Windows Cmd like below.
If you are using Windows Cmd, omit the `./`

Reply:
I got it !
I want to double check the command before editing the README.md but it's impossible because I'm using only macOS.
So, is there anyone who can check the above command ?
@seoh02h @psh2849 @Jsueeee

Reply:
Windows:
![image](https://user-images.githubusercontent.com/42669772/130014152-00264e79-cac8-42c3-933c-0e041c827a7c.png)

Windows can use without  `./` command

Reply:
Thanks ! :)
Im gonna update HOWTIBUILD.md

Reply:
Good Job :)